http-server/src/module.rs

```diff
@@ -45,7 +45,15 @@ impl RpcModule {
 			Box::new(move |id, params, tx, _| {
 				match callback(params) {
 					Ok(res) => send_response(id, tx, res),
-					Err(InvalidParams) => send_error(id, tx, JsonRpcErrorCode::InvalidParams.into()),
+					// TODO: this looks wonky...
+					Err(CallError::InvalidParams(InvalidParams)) => {
+						send_error(id, tx, JsonRpcErrorCode::InvalidParams.into())
+					}
+					Err(CallError::Failed(e)) => {
+						// TODO: do something smart(-er) with the error?
+						log::error!("Call failed with: {}", e);
+						send_error(id, tx, JsonRpcErrorCode::ServerError(CALL_EXECUTION_FAILED_CODE).into())
```

Comment:
you could include the error message in the error too if you'd like but requires some more boiler plate

Reply:
Yeah I was wondering if we had a plan for that. Do you know? Users will be sad about seeing only an error code.

Reply:
we already uses it for the context module because I assumed that only `InvalidParams` could happen here it was not considered.

so you could copy-paste that code from [here](https://github.com/paritytech/jsonrpsee/blob/master/ws-server/src/server/module.rs#L160-#L168) or extract to helper function

Reply:
Ended up copy-pasting.